docs/UserGuide.md

```diff
@@ -114,54 +125,102 @@ Parameter | Meaning
 `TAG` | This is the tag for the expenditure.
 `TYPE` | This is the sorting types.
 
-## 3. Layout
+## 3. GUI Layout
 (Contributed by Wen Hao)
+
+In this section, you will be given an introduction to the layout of NUSave's Graphical User Interface (GUI)
+which will help you better understand what each component that you observe on-screen represents.
+
+There are a total of two pages that you can navigate to when using NUSave:
+- Main page
+- Budget page
+
 ### 3.1. Main page view
+(Contributed by Wen Hao)
+
+You will be greeted by the main page upon launching NUSave. On this page, you can see a list of
+budgets that are currently stored in NUSave. Here is how the main page should look like:
+
+![Example of main page view](images/GUILayoutScreenShots/3_1_1_mainPage.png)
+
+Figure 3.1.1. Example of the main page view.
+
+Below is a table containing all the components that can be found on the main page accompanied by their respective purposes:
+
+Component | Purpose
+--------- | -------
+Command Box | A text field for you to enter your desired commands.
+Result Display | A **scrollable** panel that displays the response messages of a command after it has been executed.
+Info Box | A panel that displays the current date and time.
+Title | A text field that displays the name of the application.
+Budget Card | A card that represents a budget stored in NUSave. It contains the name of the budget as well as the total number of expenditures the budget contains.
+List View | A **scrollable** panel that can hold multiple budget cards.
 
 ### 3.2. Budget page view
+(Contributed by Wen Hao)
+
+You will be directed to a budget page when you use the [open budget command](#611-opening-a-budget-open).
+On this page, you can see a list of expenditures that are currently stored in the budget you have opened.
+Here is how a budget page should look like:
+
+![Example of budget page view](images/GUILayoutScreenShots/3_1_2_budgetPage.png)
+
+Figure 3.1.2. Example of a budget page view.
+
+Below is a table containing all the components that can be found on a budget page accompanied by their respective purposes:
+
+Component | Purpose
+--------- | -------
+Command Box | A text field for you to enter your desired commands.
+Result Display | A **scrollable** panel that displays the response messages of a command after it has been executed.
+Info Box | A panel that displays the total amount you have spent so far based on the expenditures added to the budget. It will also display the threshold of the budget if it has one.
+Title | A text field that displays the name of budget that is open.
+Expenditure Card | A card that represents an expenditure stored in a budget. It contains the name as well as the cost of the expenditure.
+List View | A **scrollable** panel that can hold multiple expenditure cards.
 
 ## 4. Quick start
 (Contributed by Wen Hao)
 
+If this is your first time using NUSave, follow these simple steps to jump straight into action:
```

Comment:
Just a nit to pick, I think it should be "jump straight into the action"

Reply:
Yes sir